src/gcs-resumable-upload/index.ts

```diff
@@ -661,10 +661,14 @@ export class Upload extends Pumpify {
 
     // Check if the offset (server) is too far behind the current stream
     if (this.offset < this.numBytesWritten) {
-      this.emit(
-        'error',
-        new RangeError('The offset is lower than the number of bytes written')
-      );
+      const delta = this.numBytesWritten - this.offset;
+      let message = 'The offset is lower than the number of bytes written. ';
```

Comment:
Looks good. Just a minor nit, could this be done as a single interpolation instead of concatenating the additional lines or do you feel it would impact readability?